File: config/30_eigen/x_rm_tmps.py
#!/usr/bin/env python

import os

tmpdir = "/tmp/cpac_eigen"

for i in xrange(1,33):
    print "========"
    
    cmd = "ssh node%i ls %s" % (i, tmpdir)
    print cmd
    os.system(cmd)
    
    cmd = "ssh node%i rm -rf %s" % (i, tmpdir)
    print cmd
    os.system(cmd)
    
    print "========"

File: config/46_xn_vmhc/x_rm_tmps.R
#!/usr/bin/env Rscript

# Subjects in working directories
for (i in 1:32) {
    cmd <- sprintf("ssh node%i ls /tmp/cpac_eigen", i)
    cat(cmd, "\n")
    system(cmd)
    
    cmd <- sprintf("ssh node%i rm -rf /tmp/cpac_eigen", i)
    cat(cmd, "\n")
    system(cmd)    
}
